.rpmlintrc

```diff
@@ -1,3 +1 @@
 addFilter('macro-in-comment')
```

Comment:
bundler_ext fails with `E: specfile-error warning: Macro expanded in comment on line 30:`, even tho this filter is still here.

should we just fix it? :)

Reply:
I wouldn't mind just fixing it.